src/operations.cpp

```diff
@@ -119,6 +128,17 @@
 #   if _POSIX_C_SOURCE < 200809L
 #     include <utime.h>
 #   endif
+#   ifdef __linux__
+#     ifdef __GLIBC__
+#       include <gnu/libc-version.h>
```

Comment:
No internal header includes, please. glibc version macros are not defined in this header anyway. Use `unistd.h` or another standard header to get them.